packages/teleterm/src/ui/TopBar/Clusters/ClustersFilterableList/ClusterItem.tsx

```diff
@@ -24,7 +23,7 @@ export function ClusterItem(props: ClusterItemProps) {
   });
 
   const LabelVersion = props.isSelected ? InvertedLabel : Label;
-  const clusterName = getClusterName(props.item);
+  const clusterName = props.item.name;
```

Comment:
Good job differentiating between places that know upfront cluster is present vs those that don't. 👍

Reply:
Could we use optional chaining like `props.item?.name`. I know it _should_ always exist but might as well throw the optional chain in there to make sure a renderer doesn't break on the off chance that name doesn't end up where we expect it.